metaflow/runtime.py

```diff
@@ -219,6 +217,13 @@ def execute(self):
                 "Workflow starting (run-id %s):" % self._run_id, system_msg=True
             )
 
+        tracing_url = tracing.get_tracing_url()
+        if tracing_url:
+            self._logger(
```

Comment:
we could print only if we are debugging tracing.

Reply:
decided to move this out of per-task level (instead put this at the same lvl like maestro links).